code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -218,7 +218,8 @@
 	new /obj/item/pinpointer/crew(src)
 	new /obj/item/clothing/shoes/laceup/digitigrade(src) //fulp
 	new /obj/item/binoculars(src)

```

Comment:
hey, you never added these binoculars to the vending machine...

also maybe consider adding some handcuffs to the vending machine as well, but they're optional, I guess

Reply:
you also never added the digitigrade laceup shoes to the vending machine

Reply:
also also, you never added a way to buy an /obj/item/vending_refill/detective from cargo

Reply:
This has been addressed.